README.md

```diff
@@ -27,7 +27,7 @@ The stackify module requires the following:
 * Puppet Agent 4.0 or later.
```

Comment:
can you bump this to 4.7.1? This is the lowest version that Vox Pupuli supports.

Reply:
Done.